### PR TITLE
Add 3 x Base Uniswap CLM Pools

### PR DESCRIPTION
**Pull request by iamjackgale**

### CODE REVIEW

**Comment by iamjackgale** [comment]
Corresponding [API PR](https://github.com/beefyfinance/beefy-api/pull/1582) ✅

File: src/config/vault/base.json [hunk 1]
```diff
@@ -62,6 +62,192 @@
       }
     ]
   },
+  {
+    "id": "uniswap-cow-base-hyUSD-USDC-rp",
```
Comment (kexleyBeefy) [inline src/config/vault/base.json:66]: Can you please make the ids lowercase, as should be the same in API

Comment (iamjackgale) [reply src/config/vault/base.json]: Think that's all done here, but the error was also in the API PR, so have put in a further PR to correct those. Thanks for spotting!

https://github.com/beefyfinance/beefy-api/pull/1585

File: src/config/vault/base.json [hunk 2]
```diff
@@ -62,6 +62,192 @@
...
+    "tokenAddress": "0xD937c1b7761f053FDDA541C731fdf255FC3222Ea",
+    "tokenDecimals": 18,
+    "tokenProviderId": "uniswap",
+    "earnedTokens": [],
+    "earnedTokenAddresses": [],
+    "earnedOracleIds": [],
+    "earnedTokenDecimals": [18],
+    "earnedToken": "rcowUniswapBasehyUSD-USDC",
+    "earnContractAddress": "0x81E4675C6A312e502eF10aA038e6159B921132C6",
+    "oracle": "lps",
```
Comment (kexleyBeefy) [inline src/config/vault/base.json:80]: missing oracleId for all of these, that's why the price is appearing as a fire emoji

Comment (iamjackgale) [reply src/config/vault/base.json:80]: Ahhhhh - thanks! Added

**Pull request merged**